README.md

```
@@ -1,276 +1,278 @@
- 1. See [CONTRIBUTING.md](CONTRIBUTING.md) for details on branching, committing, and opening PRs.  
- 2. Please review our [CODE_OF_CONDUCT.md](CODE_OF_CONDUCT.md) and [SECURITY.md](SECURITY.md) before contributing.
-
----
-
-## Resources
-
-- Community Slack: <https://slack.meshery.io/>
-
-Happy Learning!
+<div align="center">
+  <img src=".github/assets/images/meshery-logo-dark-text-side.svg" width="70%" /> <br />
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)